### ISSUE TITLE
Integrating Cient-Server example doesn't work 

### ISSUE THREAD

**Issue by kaesler** [lihaoyi/hands-on-scala-js#26]
I am working through some examples in the book and found this.  In the section "A Client-Server Setup" the build.sbt has some code which is intended to copy the output Javascript code from  the ScalaJS compilation into the resources directory of the Spray sub-project.

What task/command is supposed to trigger that copy?  It doesn't seem to work for me when I compile the JVM code.

**Comment by lihaoyi** [comment]
This line https://github.com/lihaoyi/hands-on-scala-js/blob/master/examples/crossBuilds/clientserver/build.sbt#L25 is meant to trigger the copy

Could you be bumping into this issue https://github.com/lihaoyi/hands-on-scala-js/pull/25?

**Comment by kaesler** [comment]
No I already corrected the file name....

...However, never mind. I just discovered the "copyResources" task. When I run that from the root project the js files are generated and then copied to the app/jvm/target/scala-2.11/classes/ directory,from whence the Spray app can find them. Might be a good idea to add a sentence explaining that step, or I can make a PR to that effect.

To make matters worse I am working in the Eclipse IDE which does not seem to have any way to trigger this "copyResources" task. Will work directly in SBT. I wonder if IntellIJ is better in this regard?
